Scripts/AnalyzeDatabank/calcAPL.py

```diff
@@ -1,19 +1,14 @@
 import os
 import sys
-import numpy as np
 import json
-import matplotlib.pyplot as plt
-import MDAnalysis
-import urllib.request
-import yaml
 
 ### Initializing the databank
 
 ### This is the NMRlipids databank repository path
 databankPath = '../../'
 
-sys.path.insert(1, databankPath + '/Scripts/BuildDatabank/')
-from databankLibrary import *
+sys.path.append('..')
+from DatabankLib.databankLibrary import *
```

Comment:
to have an internal consistency in importing databank related modules and methods, could you please explicitly define the imported methods here, instead of using *?

Reply:
done during style alignment.